.github/workflows/package-cli.yml

```diff
@@ -55,6 +55,17 @@ jobs:
     if: ${{ inputs.version_patch_run_id != '' && always() || success() }}
     name: 📦 Packaging CLI for linux 🐧
     runs-on: ubuntu-24.04
+    env:
+      CARGO: cargo
```

Comment:
Why not just use `cargo` directly ?

Reply:
Given `env.CARGO` is only used in the `Build CLI` step, I'd rather try to keep as much local as possible

I guess this environment variable has to do with the `echo "CARGO=cross" >> $GITHUB_ENV` in the `Use cross` step.

This issue here are:
- `CARGO` environment variable may seems like a standard environment variable that is used internally in some rust tools
- It makes the code more complex, since we have to guess what `${{ env.CARGO }}` stands for (and often got the info wrong since the variable is defined at the top of the file, so it's easy to miss the overwrite that occurs in `Use Cross`)

~So in the end I would just remove this `CARGO` variable and instead call `cross` directly~

PLOT TWIST ! `CARGO` *is* indeed [a special environ variable](https://doc.rust-lang.org/cargo/reference/environment-variables.html) (I've just discover that now ^^)

So I think we should declare this environment variable only in the step that need it (so in `Build CLI`, and we consider `Use Cross` only job is to make available in PATH the `cross` command [^1]), and with a comment explaining (something like `Tell Cargo to forward this value instead of its own auto-detected path when it builds crates and executes build scripts and external subcommands`)

[^1]: so `Use Cross` can be renamed `Install Cross`

Reply:
outdated